docker/README.md

```diff
@@ -8,5 +8,9 @@ To build this image yourself, follow these steps:
 1. Clone down the repository using `git clone https://github.com/chocolatey/choco.git`.
 1. Change directories to the root of the repository.
 1. Run the docker build command. `docker build -t mono-choco -f docker/Dockerfile.linux .` (the trailing . is important)
```

Comment:
What is the significance of `mono-choco` here?  Is this the tag? Should this be something else, like `chocolatey/choco`?

Reply:
That is the image name. The format is `<user>/<image name>:<tag>`, so that would be `chocolatey/mono-choco:latest`.

It probably should be changed to `choco:latest` or `choco:dev`. Then with the official build, you would use `chocolatey/choco:0.11.1`

I was holding off on changing it in the documentation until the naming had settled down a bit. Specifically, what tags to use.

Reply:
Ok, makes sense. Thanks for the information!